android/app/src/main/java/com/textile/textilenode/TextileImageTask.java

```diff
@@ -0,0 +1,52 @@
+package com.textile.textilenode;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.util.Base64;
+import android.widget.ImageView;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+public class TextileImageTask extends AsyncTask<Void, Void, Bitmap> {
```

Comment:
This guy allows us to load the image data from the Textile node on a background thread, and update the UI on the UI thread.

Reply:
niceee